### PR TITLE
Update virAnnot

### PR DESCRIPTION
**Pull request by marieBvr**
FOR CONTRIBUTOR:
* [x] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [x] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

File: tools/virAnnot/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- Metagenomics
+description: virAnnot wrappers
+homepage_url: https://github.com/marieBvr/virAnnot
+type: unrestricted
+long_description: VirAnnot was build to ease the assembly, blast search, taxonomic
+  annotation and OTUs assignation of viral metagenomic NGS data. It is designed to
+  identify viruses in plant metagenomic data but it can be used to assemble and annotate
+  any sequences with the NCBI taxonomy.
+name: virannot
+owner: melefebvre
```
Comment (bernt-matthias) [inline tools/virAnnot/.shed.yml:11]: owner needs to be `iuc`

**Review by bernt-matthias — COMMENTED** [review]
Can the python and R scripts be added to virannot: https://github.com/marieBvr/virAnnot/tree/master/tools? 

IMO We should try avoid having "non-trivial" scripts/software in this repo. Otherwise we "need" to give support for it.

File: tools/virAnnot/.shed.yml (hunk 1 above)
Comment (marieBvr) [reply tools/virAnnot/.shed.yml]: Yes, the python and R scripts can be added. This is to fix some part of the code that was causing errors.

**Comment by bgruening** [comment]
Can we close this PR? We merged https://github.com/galaxyproject/tools-iuc/pull/5482 and now here is a large merge conflict.

**Comment by marieBvr** [comment]
My apologies, I must not have followed the correct procedure for making this fix. 
Yes, the PR can be closed.

**Comment by marieBvr** [comment]
Hi @bgruening, @bernt-matthias,
Is there anything I can do to improve this PR?

Thanks

**Comment by bgruening** [comment]
@marieBvr you would need to rebase this PR against the main git branch. GitHub is showing a lot of merge conflicts at the moment.

**Pull request closed without merging**